superset/constants.py

```diff
@@ -58,4 +59,4 @@ class RouteMethod:  # pylint: disable=too-few-public-methods
     # Commonly used sets
     CRUD_SET = {ADD, LIST, EDIT, DELETE, ACTION_POST}
     RELATED_VIEW_SET = {ADD, LIST, EDIT, DELETE}
-    REST_MODEL_VIEW_CRUD_SET = {DELETE, GET, GET_LIST, POST, PUT}
+    REST_MODEL_VIEW_CRUD_SET = {DELETE, GET, GET_LIST, POST, PUT, INFO}
```

Comment:
I'm a little concerned with this pattern overall. Altering these "defaults" down the road could lead to unintended side effects.

Reply:
We can remove it from the default and enable the info endpoint on a per-case basis. That was the original approach: https://github.com/apache/incubator-superset/pull/9015/commits/a5488e2aad8cc7b0daa95bc9382697dcc07a51b5
but it seems like we'll need to info endpoint in most cases